microsetta_interface/implementation.py

```diff
@@ -145,7 +145,7 @@ class Source:
 client_state = RedisCache()
 
 # Countries allowed to view the contents of the My Kits tab
-KITS_TAB_WHITELIST = {'US', 'ES', 'JP'}
+KITS_TAB_WHITELIST = {'US', 'ES', 'JP', 'MX', 'GB'}
```

Comment:
no SP?

Reply:
Spain is ES, unless there's another country I'm missing?

Reply:
lol ya